tests/test_smoke.py

```diff
@@ -288,6 +292,36 @@ def test_propagate_non_default_compiler(self, capfd, chdir_build):
         assert "The C compiler is not defined." not in err
         assert 'tools.build:compiler_executables={"c":"/usr/bin/clang","cpp":"/usr/bin/clang++"}' in out
 
+class TestProfileCustomization:
+    def test_profile_defults(self, capfd, chdir_build):
+        """Test the defaults passed for host and build profiles"""
+        run(f"cmake --fresh .. -DCMAKE_PROJECT_TOP_LEVEL_INCLUDES=conan_provider.cmake -DCMAKE_BUILD_TYPE=Release", check=True)
+        builddir = str(Path.cwd()).replace('\\','/')
+        out, _ = capfd.readouterr()
+        assert f"--profile:host={builddir}/conan_host_profile" in out
```

Comment:
It'd be enough to check that the pattern is there instead of getting the whole path:

```python
        out, _ = capfd.readouterr()
        assert re.search("--profile:host=.*/build/conan_host_profile", out)  # buildidr
```

Reply:
yeah, this is a lot better!